### PR TITLE
update steam multi tooltips

### PR DESCRIPTION
**Pull request by chochem**
removes some outdated tier dependence for usage, makes them a bit shorter to actually fit the line, adds the missing info for the mixer, and makes sure they are all consistent with each other.

### CODE REVIEW

File: src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/steam/MTESteamCentrifuge.java [hunk 1]
```diff
@@ -328,9 +328,8 @@ protected MultiblockTooltipBuilder createTooltip() {
         MultiblockTooltipBuilder tt = new MultiblockTooltipBuilder();
         tt.addMachineType(getMachineType())
             .addInfo("Controller Block for the Steam Centrifuge")
-            .addInfo("33.3% faster than a single block steam machine would run.")
-            .addInfo(
-                "On Tier 1, it uses only 66.6% of the steam/s required compared to what a single block steam machine would use.")
+            .addInfo("33.3% faster than a single block steam machine would run")
```
Comment (Elisis) [inline src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/steam/MTESteamCentrifuge.java:331]: Maybe specifically mention the SB centrifuge, so it's consistent with the other tooltips?

**Comment by OneEyeMaker** [comment]
Also check description of Primitive Water Pump. There should be Output Hatch instead of Input in description.

File: src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/steam/MTESteamCentrifuge.java (hunk 1 above)
Comment (chochem) [reply src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/steam/MTESteamCentrifuge.java:331]: there is none. thats why those are formulated differently

**Comment by chochem** [comment]
> Also check description of Primitive Water Pump. There should be Output Hatch instead of Input in description.

done.

**Review by OneEyeMaker — APPROVED** [review]

**Pull request merged**